mentoring/tests/integration/test_mcq.py

```diff
@@ -231,3 +260,11 @@ def test_questionnaire_html_choices(self, page):
         self.wait_until_disabled(submit)
 
         self.assertIn('Congratulations!', messages.text)
+
+
+@mock.patch.object(MentoringBlock, 'get_theme', mock.Mock(return_value={'package': None, 'locations': []}))
```

Comment:
@singingwolfboy Without the `mock.Mock` the decorator would pass the automatically created mock object to the test functions, which would break my tests. This is mentioned in the docs in the three args vs two args section: http://www.voidspace.org.uk/python/mock/patch.html#patch-object